### PR TITLE
Adjust configuration to new config sample

### PR DESCRIPTION
**Pull request by michaelsauter**
Uses the new, unified files and adds PROV_APP_ prefix to the params.

Relates to https://github.com/opendevstack/ods-core/issues/219.

I left the Rundeck stuff in on purpose ... just want to make it "work", we can kill all the Rundeck things later.

### CODE REVIEW

File: ocp-config/prov-app/cm.yml [hunk 1]
```diff
@@ -131,24 +131,24 @@ objects:
...
-- name: JIRA_URI
+- name: PROV_APP_JIRA_URI
   required: true
   description: the URI of JIRA used to create new jira projects  
-- name: CONFLUENCE_URI
+- name: PROV_APP_CONFLUENCE_URI
   required: true
   description: the URI of confluence used to create new confluence spaces
-- name: BITBUCKET_URI
+- name: PROV_APP_BITBUCKET_URI
```
Comment (clemensutschig) [inline ocp-config/prov-app/cm.yml]: @michaelsauter bitbucket URI is used in many placed, or? .. if so - this should not be "prov_app" :)

Comment (michaelsauter) [reply ocp-config/prov-app/cm.yml]: Yes it is. BUT I was not sure if I should use the "globally configured" one here as we have two instances (prov-dev and prov-test) and one might want to configure them differently?

IMHO eventually ODS should ship with one central ProvApp instance in the `cd` namespace (meaning to treat it similar to Nexus/SonarQube). If we were to do that for e.g. v3/v4 then I would propose to just use the globally configured params then. For now though I'd stick with separate configs.

Do you agree or would you rather have a fixed single Atlassian stack for both prov-dev and prov-test instances?

Comment (clemensutschig) [reply ocp-config/prov-app/cm.yml]: I would use the global one ... otherwise - you need to introduce a double pair (dev / test :))

Comment (tjaeschke) [reply ocp-config/prov-app/cm.yml]: I'm with @michaelsauter.

Comment (clemensutschig) [reply ocp-config/prov-app/cm.yml]: ok  then we go with it :)

**Pull request merged**